service/cert-signer/cert_signer.go

```
@@ -52,6 +57,34 @@ type certSigner struct {
 }
 
 func (cs *certSigner) Issue(config spec.IssueConfig) (spec.IssueResponse, error) {
+	var roleService role.Service
+	var err error
+	{
+		roleServiceConfig := role.DefaultConfig()
+		roleServiceConfig.VaultClient = cs.VaultClient
+		roleServiceConfig.PKIMountpoint = fmt.Sprintf("pki-%s", config.ClusterID)
+		roleService, err = role.New(roleServiceConfig)
+		if err != nil {
+			log.Fatalf("%#v\n", microerror.Mask(err))
```

Please return the error properly to the caller.